packages/ui/src/components/SearchEdit.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import { translate } from '@anticrm/platform'
+
+  import { createEventDispatcher } from 'svelte'
+  import ui from '../plugin'
+  import EditWithIcon from './EditWithIcon.svelte'
+  import IconSearch from './icons/Search.svelte'
+
+  export let value: string = ''
+
+  $: _search = value
+  const dispatch = createEventDispatcher()
+  let placeholder = ''
+
+  translate(ui.string.Search, {}).then((v) => {
+    placeholder = v
+  })
+</script>
+
+<EditWithIcon
+  icon={IconSearch}
+  placeholder={placeholder}
+  bind:value={_search}
+  on:change={() => {
+    if (_search === '') {
+      value = ''
+      dispatch('change', '')
+    }
+  }}
+  on:input={() => {
+    if (_search === '') {
+      value = ''
+      dispatch('change', '')
```

Comment:
Why we need clear search when input?

Reply:
input is handled for any change, like copy/paste. And for empty case I've specially added to inform about changes with no waiting for pressing Enter